.gitignore

```diff
@@ -23,3 +23,4 @@ test.html
 
 # shh
 *.ignore.*
+.vscode/taskmarks.json
```

Comment:
do you think it'll be worth a blanket ignore on the entire `.vscode` directory?

Reply:
personally i think it's not worth really pushing stuff in the .vscode folder, but idr what the fuck it does. gonna have to leave this to my boy ackwell

Reply:
It might be worth ignoring everything in .vscode, and adding exceptions for files we explicitly want. Stuff like the extension recommendations is handy IMO.